packages/swap/src/handlers/openSwapDepositChannel.ts

```diff
@@ -51,6 +51,7 @@ export default async function openSwapDepositChannel(
       create: {
         ...rest,
         depositAddress,
+        srcChainExpiryBlock: sourceChainExpiryBlock,
```

Comment:
would have expected this to trigger a test, but i guess we dont test the api? 🤔

Reply:
this was not tested by the api tests, however I've added a simple unit test for it now